community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: tidb-operator.1.0.0-beta1
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+        [  
+          {
+            "apiVersion": "pingcap.com/v1alpha1",
+            "kind": "TidbCluster",
+            "metadata": {
+              "name": "demo",
+              "namespace": "tidb1"
+              },
+            "spec": {
+              "pd": {
+                  "image": "pingcap/pd:v2.1.0",
+                  "imagePullPolicy": "IfNotPresent",
+                  "limits": {},
+                  "nodeSelectorRequired": "true",
```

Comment:
Getting this error when running the example CR: 
v1alpha1.PDSpec.NodeSelectorRequired: ReadBool: expect t or f, but found ", 
Is this implying that this line should have a t or f instead of "true"?